### PR TITLE
Waiting for KafkaFeature during topic creation

### PR DESCRIPTION
**Pull request by pitagoras3**
Previously, `KafkaBrokerTopicManagement` was not waiting for topic created confirmation from Kafka. Now we can handle topic creation failures, and return correct error to end user.

### CODE REVIEW

File: hermes-management/src/main/java/pl/allegro/tech/hermes/management/infrastructure/kafka/service/KafkaBrokerTopicManagement.java [hunk 1]
```diff
@@ -19,7 +20,10 @@
 import java.util.HashMap;
 import java.util.Map;
 import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
 import java.util.stream.Collectors;
+import java.util.stream.Stream;
```
Comment (szczygiel-m) [inline hermes-management/src/main/java/pl/allegro/tech/hermes/management/infrastructure/kafka/service/KafkaBrokerTopicManagement.java:26]: You don't use this imports

File: hermes-management/src/main/java/pl/allegro/tech/hermes/management/infrastructure/kafka/service/KafkaBrokerTopicManagement.java [hunk 2]
```diff
@@ -39,15 +43,21 @@ public KafkaBrokerTopicManagement(TopicProperties topicProperties, AdminClient k
...
                         ).configs(config)
                 ))
-        );
+        ).forEach(result -> {
+            try {
+                result.all().get();
+            } catch (InterruptedException | ExecutionException e) {
+                throw new BrokersClusterCommunicationException(e);
+            }
+        });
```
Comment (piotrrzysko) [inline hermes-management/src/main/java/pl/allegro/tech/hermes/management/infrastructure/kafka/service/KafkaBrokerTopicManagement.java:60]: There are similar problems in a few other places in this class:
* line 65
* line 74
* line 124

File: hermes-management/src/main/java/pl/allegro/tech/hermes/management/infrastructure/kafka/service/KafkaBrokerTopicManagement.java (hunk 1 above)
Comment (pitagoras3) [reply hermes-management/src/main/java/pl/allegro/tech/hermes/management/infrastructure/kafka/service/KafkaBrokerTopicManagement.java]: Thanks, that is leftover after experiments. Removed!

File: hermes-management/src/main/java/pl/allegro/tech/hermes/management/infrastructure/kafka/service/KafkaBrokerTopicManagement.java (hunk 2 above)
Comment (pitagoras3) [reply hermes-management/src/main/java/pl/allegro/tech/hermes/management/infrastructure/kafka/service/KafkaBrokerTopicManagement.java]: Thanks for this catch! I've extracted `waitForKafkaFuture` method, which now is applied to all places you mentioned.

**Review by szczygiel-m — APPROVED** [review]

**Review by faderskd — APPROVED** [review]

**Review by piotrrzysko — APPROVED** [review]

**Pull request merged**